### DISCUSSION TITLE
[Tracefile] page.percySnapshot() does not appear in html or tracefile

### DISCUSSION THREAD

**Discussion by unlikelyzero** [percy/percy-playwright#130] [Ideas]
## The problem

We're debugging some problems in our CI related to percy and noticed that the percySnapshot() commands are not included in the HTML report nor the tracefile. https://github.com/nasa/openmct/issues/5742

## Environment

- Node version: 16
- `@percy/cli` version: 1.10.0
- Version of Percy SDK you’re using:
- If needed, a build or snapshot ID:
- OS version: focal ubuntu
- Type of shell command-line [interface]:

## Debug logs

https://circleci.com/api/v1.1/project/github/nasa/openmct/49758/output/106/0?file=true&allocation-id=631b813cddd6ab006df32154-0-build%2F3ADEF78B

## Code to reproduce issue

This is all open source. You can see the html report and tracefiles on this run

https://app.circleci.com/pipelines/github/nasa/openmct/13274/workflows/44cd228c-30b6-4fc2-b237-40e08c290419/jobs/49758/steps

HTML Report
https://output.circle-artifacts.com/output/job/fe2433fc-3e3b-4702-8dce-d0f52b39d6f2/artifacts/0/html-test-results/index.html

Tracefile
https://output.circle-artifacts.com/output/job/fe2433fc-3e3b-4702-8dce-d0f52b39d6f2/artifacts/0/html-test-results/data/99a21c6592bc4170505b7f7d6766d65fed3cbe9c.zip

**Comment by Robdel12** [comment]
👋🏼 Hey @unlikelyzero! This makes sense, Percy isn't really calling any assertions or anything like that. You can _technically_ see Percy in there when you look at the page.evaluates (since that's pretty much all the SDK does): 
<img width="686" alt="image" src="https://user-images.githubusercontent.com/2072894/189435348-33caccbe-f851-45c0-973e-45c399b89e49.png">

**Reply by unlikelyzero** [reply]
@Robdel12 got it. I've opened a ticket in the official playwright issue tracker. https://github.com/microsoft/playwright/issues/17226

Ideally, we'd have a way to "register" this alias with the reporter.
